.gitignore

```diff
@@ -31,3 +31,6 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+.ruby-gemset
```

Comment:
Not a super strong opinion, but I think I would prefer not to add this to the repo but rather configure it on our global `.gitignore` instead since it is more of a `rvm` specific file

Reply:
I use rvm on my local machine, so I can't just add to the global .gitignore in my machine.
I need to do on the repo itself.

But I found a way to do that (locally on my machine, on a repo based) a few months ago, while contributing to another OSS repo.
I will need to try to find that again and remember what I did, so if you prefer I can revert it.